browser/js/common/directives/graphs/add-graph.directive.js

```diff
@@ -0,0 +1,77 @@
+app.directive('addGraph', function($http) {
+    return {
+        restrict: 'E',
+        templateUrl: 'js/common/directives/graphs/add-graph.directive.html',
+        scope: {
+            data: "=",
+            info: "=",
```

Comment:
I'm not seeing `scope.info` being used anywhere in this directive or its template. Is this something you're intending to use down the road?